fix update version for create; add type linting to docs and create (#317)
open-wc/open-wc: docs/developing/types.md

## Open Web Components Types

For most of our products we do offer types via JSDocs.

> In order to utalize them you will need to set them up for your project.

Reviewer: ```suggestion
In order to utilize types, you will need to set them up for your project.
```

> ### Why should I care?

Reviewer: ```suggestion
### Why types?
```

> Simply speaking this will improve your developer experience by giving you

Reviewer: ```suggestion
Types can help improve your developer experience by giving you:
```

> - Nice intellisense for @open-wc tools/helpers

Reviewer: ```suggestion
- Awesome intellisense for @open-wc tools/helpers
```

> - Direct go to the source of @open-wc code via F12 (in vs code)

Reviewer: ```suggestion
- Ability to jump directly to the source of @open-wc code via F12 (in vs code)
```
